Meetings/2019-12-03.md

```diff
@@ -1,25 +1,44 @@
 # TSC Meeting 2019 December 3rd @12PM MST.
-- Zoom Link: 
+- Zoom Link: https://zoom.us/j/662376664
```

Comment:
Do you need a new link?

Reply:
No, I put a new one in the January page and on the Google Calendar.